File: c/Lista3/Desafio7.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

main(){
    
    float salario, PR, SalarioFinal;
    
printf("Digite o valor do salário: ");
scanf("%f", &salario);
    
printf("Digite o percentual de reajuste: ");
scanf("%f", &PR);
    
SalarioFinal = (salario * PR) / 100;
printf("O valor retirado do seu salário é: %3.0f", SalarioFinal);
    
}

File: js/lista3/Desafio7.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
}); 

/*
Desafio7- Escreva um algoritmo para ler o salário mensal e o percentual de reajuste:

Calcular o salário final do funcionário;

Escrever o valor do novo salário.
*/

rl.question('Digite o valor do salário mensal: ', (salarioMensal) => {
    rl.question('Digite o valor do reajuste: ', (reajuste) => {

let salarioFinal = salarioMensal;
let novoSalario = salarioFinal + salarioMensal;
    
        console.log(`O salário do funcionário é: ${salarioFinal} e o novo salário é: ${novoSalario}`);
      rl.close();
    });
  });
    